sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/TokenIssuanceStart/Data/AuthenticationEventContextUser.cs

```diff
@@ -15,10 +15,10 @@ public partial class AuthenticationEventContextUser
         [JsonPropertyName("companyName")]
         public string CompanyName { get; set; }
 
-        /// <summary>Gets the country.</summary>
-        /// <value>The country.</value>
-        [JsonPropertyName("country")]
-        public string Country { get; set; }
+        /// <summary> Gets or sets the created date time.</summary>
+        /// <value> The created date time. </value>
```

Comment:
```suggestion
        /// <summary>Gets or sets the created date time.</summary>
        /// <value>The created date time.</value>
```

The spacing that was added here makes the formatting inconsistent with the other members of this file.